### PR TITLE
📚  add filter docs

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

Adds documentation describing the desired filtering approach for tumor only.

Part of https://d3b.atlassian.net/browse/BIXU-2125

## Type of change

Please delete options that are not relevant.

- [x]  documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Nothing to test

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have committed any related changes to the PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,20 +48,22 @@ Most inputs have recommended values that should auto import both files and param
  - `genomic_hotspots`: tert.bed # bed file with TERT gene promoter region
  - `protein_snv_hotspots`: protein_snv_cancer_hotspots_v2.ENS105_liftover.tsv
  - `protein_indel_hotspots`: protein_indel_cancer_hotspots_v2.ENS105_liftover.tsv
- - `echtvar_anno_zips`: gnomad.v3.1.1.custom.echtvar.zip
+ - `echtvar_anno_zips`: gnom ad.v3.1.1.custom.echtvar.zip
```
Comment (migbro) [inline README.md:51]: I think you added a space by  accident

File: docs/MUTECT2_TUMOR_ONLY_FILTERING.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+## Recommendations from Benchmarking
+
+Tumor-only calling with Mutect2 is inherently very noisy. Therefore, most of our recommendations revolve around reducing
+the noise where possible. These recommendations are:
+- Restrict the callable regions with a blacklist and GDC Panel of Normals (PON)
+- Filter low support reads:
+   - Allele Depth (AD) > 0: WGS uninformative reads
+   - Variant Allele Frequency (VAF) > 1%: WXS noise
+- Filter potential germline variants with gnomAD AF < 0.00003
```
Comment (migbro) [inline docs/MUTECT2_TUMOR_ONLY_FILTERING.md:57]: So, to be clear, not `0.003`, but `0.0003`? We were using I think `0.001` before which was from somatic criteria

**Review by sickler-alex — APPROVED** [review]
Very thorough. Looks good to me.

File: docs/MUTECT2_TUMOR_ONLY_FILTERING.md (hunk 2 above)
Comment (migbro) [reply docs/MUTECT2_TUMOR_ONLY_FILTERING.md:57]: NM, I reads further, and you really meant this

**Review by migbro — COMMENTED** [review]
Documentation is nice and helpful. Looking food so far, but I anticipate more coming

File: docs/MUTECT2_TUMOR_ONLY_FILTERING.md (hunk 2 above)
Comment (dmiller15) [reply docs/MUTECT2_TUMOR_ONLY_FILTERING.md:57]: I'll try and clear this up. I guess what I'm trying to say is "get rid of variants that have gnomAD AF > 0.00003"

Comment (dmiller15) [reply docs/MUTECT2_TUMOR_ONLY_FILTERING.md]: Done

**Review by migbro — APPROVED** [review]
Looks good!

**Pull request merged**